TFUtil.py

```diff
@@ -91,6 +92,7 @@ def __init__(self, name,
       The size is always a tensor of shape (batch,), i.e. the size can be different for each sequence in a batch.
     :param bool available_for_inference: e.g. the extern data "classes" is usually not available for inference
     :param str|dict[str]|GeneratingDataset.Vocabulary|None vocab:
+    :param str|"UNK" unknown_label: unknown word label for the optional vocabulary
```

Comment:
"`"UNK"`" is not a type. Descriptions for default values or so, should go into the description (after the `:`). But anyway, this argument should be removed.